internal/casing/camel.go

```diff
@@ -50,6 +50,24 @@ func Camel(s string) string {
 	return string(t)
 }
```

Comment:
Hm, do we still need the old function? We should probably always use the new one?

Reply:
I expect their logic to be roughly the same, though it might be worth using the new one anyway. We should probably always be consistent with what APIv2 uses (the new repo).

Reply:
I don't think we need to make any changes here, but there may be a subtle bug hidden somewhere here that will eventually rear its head. By APIv2 I mean the Go protobuf stack in google.golang.org/protobuf. That's the "new" stack. See https://blog.golang.org/protobuf-apiv2 for more information. The grpc-gateway v2 is built on APIv2.